src/Auth/Protocol/OAuth.elm

```diff
@@ -105,9 +107,16 @@ initiateSignin sessionId baseUrl isDev config asBackendMsg now backendModel =
 generateSigninUrl : Url -> Auth.Common.State -> Auth.Common.ConfigurationOAuth frontendMsg backendMsg frontendModel backendModel -> Url
 generateSigninUrl baseUrl state configuration =
     let
+        queryAdjustedUrl =
+            if configuration.allowLoginQueryParameters then
+                baseUrl
+
+            else
+                { baseUrl | query = Nothing }
```

Comment:
I'm not sure if I'm a fan of this being transparently dropped. Maybe there's a nice way of lifting this up to the config level in future.

Reply:
IIRC the issue was that with google login query parameters can't be used, so added this configuration option and set it as false for google login.

Reply:
Added a code comment to this effect